### DISCUSSION TITLE
【GoProMAX】 finding feature to lock mode

### DISCUSSION THREAD

**Discussion by curious-jp** [gopro/labs#142] [Q&A]
Hello everyone,
I'm looking for feature that could lock the mode of GoProMAX.
I can find "Archive mode" that let us lock the mode but I want my GoProMAX to stay on even after I stop taking video.

I need my go pro to be on even if it's not on shooting.
(this is because I want to access GoProMAX's storage through wi-fi feature)

Is there any feature that enable me to full fill this requirements?

**Comment by dnewman-gpsw** [comment]
Is disabling auto off insufficient?  Please explain why you need a locked camera that doesn't turn off.

**Reply by curious-jp** [reply]
I want my GoProMAX turned on because I'm now making a smartphone app that access automatically to GoProMAX and 
transfer video data to use it in my service.

It is not me who's going to capture the video and there might also be some elderly peoples too.
That is why I want my GoProMAX to not change mode(such as time-lapse or photo mode)but only take 360video and also won't shutdown right after stop shooting.

**Reply by dnewman-gpsw** [reply]
That will be difficult. When will the camera know to shutdown?  Is it to be externally powered?

**Reply by curious-jp** [reply]
GoPro will work on battery while shooting videos but it will be externally powered for rest of the time.
After videos being uploaded to a smart phone, GoPro turns off by auto off feature so it could save battery lifespan.
